### ISSUE TITLE
Thick black or noisy border in controller frame

### ISSUE THREAD

**Issue by apoorvkh** [allenai/ai2thor#776]
I am observing an error in which images appear truncated, with a black or noisy border covering 20% of the image from the bottom. Reproduced in versions 3.2.0 and 3.2.1.

https://colab.research.google.com/drive/135LjFO26SBEUWr3NRNbw27XyCfkLJVva?usp=sharing

**Comment by mattdeitke** [comment]
I think this is the same as https://github.com/allenai/ai2thor/issues/531

Which machine are you using?

**Comment by apoorvkh** [comment]
Interesting, I am using vision-server6.

Another theory: perhaps this is happening because I have set `y` in the agent's position to a non-standard number in `TeleportFull`?

**Comment by mattdeitke** [comment]
Looks like it is working as expected for me with `ai2thor-colab` (i.e., no black bars)

https://colab.research.google.com/drive/17SICesXUvbO3wz4yrzsmOIxSx5dnKmWX?usp=sharing

So it's almost certainly the #531 issue.

It has to do with this line in the start x server script, which sets the size of the display to 1024x768. https://github.com/allenai/ai2thor/pull/763/files#diff-23e914ccfcf8ffffeec891b6e392323dae0db0d07b4df6ceef1cce9865e78dfdR152

Basically one has to stop the current xserver, change this line to something like 2000x2000, and then restart the xserver.

**Comment by apoorvkh** [comment]
Got it, thanks Matt!
